README.md

```diff
@@ -0,0 +1,85 @@
+# WebpackEncoreBundle: Symfony integration with Webpack Encore!
+
+This bundle allows you to use the `splitEntryChunks()` feature
+from [Webpack Encore](https://symfony.com/doc/current/frontend.html)
+by reading an `entrypoins.json` file and helping you render all of
+the dynamic `script` and `link` tags needed.
+
+Install the bundle with:
+
+```
+composer require symfony/webpack-encore-bundle
+```
+
+## Configuration
+
+If you're using Symfony Flex, you're done! The recipe will
+pre-configure everything you need in the `config/packages/webpack_encore.yaml`
+file:
+
+```yaml
+# config/packages/webpack_encore.yaml
+webpack_encore:
+    # The path where Encore is building the assets - i.e. Encore.setOutputPath()
+    output_path: '%kernel.public_dir%/build'
+
+    # The public prefix to your assets that you normally use with the asset() function (e.g. build/) -
+    # should match the "setManifestKeyPrefix()" value in webpack.config.js, if set.
+    asset_path_prefix: 'build/'
```

Comment:
Not sure if it still needs to be added, but this option isn't supported with the current configuration